source/features/resolve-comments.js

```diff
@@ -0,0 +1,74 @@
+import select from 'select-dom';
+import {h} from 'dom-chef';
+import {getUsername} from '../libs/utils';
+import * as icons from '../libs/icons';
+
+const RESOLVED_TEXT_REGEX = /^Comment resolved by @([a-zA-Z0-9-]+)/;
+
+export default () => {
+  for (const comment of select.all('.comment.timeline-comment')) {
```

Comment:
What do you think about selecting the "Edited" element (so we have to parse fewer elements) and then using `edited.closest('.comment')` to get the `comment` variable?